SPM Example/PortalSwift/ViewController.swift

```diff
@@ -724,8 +724,8 @@ class ViewController: UIViewController, UITextFieldDelegate {
       try portal.setGDriveView(self)
       try portal.setPasskeyAuthenticationAnchor(self.view.window!)
       try portal.setPasskeyConfiguration(relyingParty: config.relyingParty, webAuthnHost: config.webAuthnHost)
-
-      self.logger.info("ViewController.registerPortal() - Portal API Key: \(portal.apiKey)")
+      // The apikey is private within the Portal SDK class, so it must not be accessible from outside.
```

Comment:
can you add a print statement from the response for portalex that will include the api key? i use it quite often when developing to test things out so removing this print statement will be a bummer. I believe it is on sign in or sign up where we get a response from the portal ex (custodian url) server with the api that we can print